pkg/compiler/types.go

```diff
@@ -645,6 +645,16 @@ var typeString = &typeDesc{
 				NoZ:        t.Ident == stringnoz,
 			}
 		}
+		if len(args) > 0 && t.Ident == "glob" {
+			base.TypeSize = comp.stringSize(t, args)
```

Comment:
This does not look correct. We can't predict size of the type at this point (we don't know actual files). It should be variable-size always I think.